rolling/ci-clang-tidy.yaml

```diff
@@ -0,0 +1,95 @@
+%YAML 1.1
+# ROS buildfarm ci-build file
+---
+build_environment_variables:
+  ROS_PYTHON_VERSION: '3'
+  RTI_NC_LICENSE_ACCEPTED: 'yes'
+build_tool: colcon
+build_tool_args: '--cmake-args -DCMAKE_BUILD_TYPE=Debug -DSKIP_MULTI_RMW_TESTS=1 -DCMAKE_EXPORT_COMPILE_COMMANDS=ON --no-warn-unused-cli'
+build_tool_test_args: '--ctest-args -L clang_tidy --pytest-args -m clang_tidy'
+install_packages:
+- default-jdk # for CycloneDDS
+- libasio-dev # for FastRTPS
+- libtinyxml2-dev # for FastRTPS
+- maven # for CycloneDDS
+jenkins_job_label: ci-agent
+jenkins_job_priority: 50
+jenkins_job_schedule: H H * * 0
```

Comment:
This job is scheduled to run once a week on Sunday, in Jenkins crontab syntax `H` will distribute the start time arbitrarily within the period so `H H * * 0` should run once at a random minute of a random hour on Sunday.